src/helpers/language.ts

```diff
@@ -1,17 +1,44 @@
 ﻿import MyDataHelps from "@careevolution/mydatahelps-js"
 import englishStrings from "./strings-en"
 import spanishStrings from "./strings-es"
+import dutchStrings from "./strings-nl"
+import germanStrings from "./strings-de"
+import frenchStrings from "./strings-fr"
+import portugueseStrings from "./strings-pt"
+import italianStrings from "./strings-it"
+import polishStrings from "./strings-pl"
+
+export type Language = "" | "en" | "es" | "nl" | "de" | "fr" | "pt" | "it" | "pl"
 
 export function language(key: string) {
-	if (!MyDataHelps.getCurrentLanguage().toLowerCase().startsWith("es")) {
-		return englishStrings[key];
-	} else {
-		if (spanishStrings[key] != null) {
-			return spanishStrings[key];
-		} else {
-			return englishStrings[key];
-		}
-	}
+	const currentLanguage: Language = getLanguageFromIso(MyDataHelps.getCurrentLanguage());
+
+	if (currentLanguage == "en") return englishStrings[key];
+	if (currentLanguage == "es") return spanishStrings[key];
+	if (currentLanguage == "nl") return dutchStrings[key];
+	if (currentLanguage == "de") return germanStrings[key];
+	if (currentLanguage == "fr") return frenchStrings[key];
+	if (currentLanguage == "pt") return portugueseStrings[key];
+	if (currentLanguage == "it") return italianStrings[key];
+	if (currentLanguage == "pl") return polishStrings[key];
+
+	return englishStrings[key];
+}
+
+export function getLanguageFromIso(language: string): Language {
+    if (language.length < 2) return "";
+
+    var beginningOfLanguage = language.slice(0, 2);
+    if (beginningOfLanguage == "en") return "en";
```

Comment:
Feels like there may be some ways of simplifying this by making Language an enum so you could enumerate the values and check to see if beginningOfLanguage is in that enum, otherwise return "" or something?  

Anyways, not a big deal, but it would be nice to have to touch fewer places when adding more languages in the future

Reply:
I do think long term once we get >10 languages this will be the way to go.